.editorconfig

```diff
@@ -8,6 +8,3 @@ indent_size = 4
 indent_style = space
 max_line_length = 80
 trim_trailing_whitespace = true
-
-[Makefile]
-indent_style = tab
```

Comment:
Why do you want the above removed? You introduced it in this PR, I extracted it into a separate commit because I didn't want to pollute this PR with unrelated changes, and I merged it into master.